### PR TITLE
Vertex AI Experiments - 	Model training - Notebook review 

### PR DESCRIPTION
**Pull request by inardini**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by inardini** [comment]
ready to be reviewed

**Comment by inardini** [comment]
Ready to be reviewed

File: notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb [hunk 1]
```diff
@@ -174,9 +174,8 @@
         "if IS_WORKBENCH_NOTEBOOK:\n",
```
Comment (gericdong) [inline notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb:174]: <p>Is there a reason not to clean up the bucket by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Comment (inardini) [reply notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb:174]: <p>That's a good question. I think it is as a precaution. Andy suggested to set it as False. </p>

**Review by gericdong — APPROVED** [review]
LGTM

**Pull request merged**